#160283217 Update readme

tutugodfrey opened the pull request:
#### What does this PR do?
- Update readme

#### Description of Task to be completed?
- Add description of the application
- outline how the application can be started
- add the link to API Docs for building the application
- Add a table of the path/ routes available

#### How should this be manually tested?
- N/A

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
#160283217 

#### Screenshots (if appropriate)
-  N/A

#### Questions:
- N/A

rhenshaw56 commented on README.md:
```diff
@@ -1,2 +1,54 @@
+## Starting the application in development
+Clone the application to your local machine. In your terminal run `npm install` to install dependencies. Then run `npm run dev` to start the application. The application will start and open in your default browser.
+
+## Relevant urls
```
you don't need this api methods in here, take them off

rhenshaw56 reviewed (changes requested):
The usual template for writing a readme usually has  the following headers.
FEATURES|GETTING STARTED|TECHNOLOGIES|CONTRIBUTIONS

I see none here

JCanaks commented on README.md:
```diff
@@ -1,2 +1,54 @@
+# rivendell-ah-client Author's Haven
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging)
+
+Authors Haven is am online application that bring together a community of authors and readers. It allows Authors to create and post articles based on topic of interest or popular demands. Reader can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
```
correct typos  on  description it should be `Author's haven is 'an' online application that 'brings' together a community of authors .....It allows 'authors' (lowercase a)...... based on 'their' .....or popular 'demand' (singular). 'Readers' (plural)`

JCanaks commented on README.md:
```diff
@@ -1,2 +1,54 @@
+Authors Haven is am online application that bring together a community of authors and readers. It allows Authors to create and post articles based on topic of interest or popular demands. Reader can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
+- Users are able to give ratings for articles
+- Users can like /dislike articles
+- Users are able to mark articles as favorite
```
simplify to  `users can mark an article as a favorite`

JCanaks commented on README.md:
```diff
@@ -1,2 +1,54 @@
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging)
+
+Authors Haven is am online application that bring together a community of authors and readers. It allows Authors to create and post articles based on topic of interest or popular demands. Reader can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
+- Users are able to give ratings for articles
```
simplify to  `Users can rate an article`

tutugodfrey committed:
chore(readme): update readme

- add description for the app
- add feature supported
- add instructor to start the app in dev mode
- add link to backend doc
- add contributors

JCanaks commented on README.md:
```diff
@@ -1,2 +1,61 @@
 # rivendell-ah-client
 [![CircleCI](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging.svg?style=svg)](https://circleci.com/gh/andela/rivendell-ah-client/tree/staging) [![Test Coverage](https://api.codeclimate.com/v1/badges/752141b5976d9ebab531/test_coverage)](https://codeclimate.com/github/andela/rivendell-ah-client/test_coverage)
+
+Authors Haven is online application that brings together a community of authors and readers. It allows authors to create and post articles based on their topic of interest or popular demand. Readers can search for articles to read. Readers can also follow authors if they like their articles. Other features supported by the application includes: 
```
Author's haven is 'an' online

JCanaks commented on README.md:
```diff
@@ -1,2 +1,61 @@
+- Users can register with Oauth services (facebook, linkedin, twitter)
+- Users can rate articles
+- Users can like /dislike articles
+- Users Can mark articles as favorite
```
as 'a' favorite
